### ISSUE TITLE
Error in .order_seqlevels(chrom_sizes[, "chrom"]) :  !anyNA(m32) is not TRUE

### ISSUE THREAD

**Issue by sanchit-saini** [Bioconductor/GenomeInfoDb#32]
```R
> suppressPackageStartupMessages(library(GenomeInfoDb))
> Seqinfo(genome="hg38")
Error in .order_seqlevels(chrom_sizes[, "chrom"]) :
  !anyNA(m32) is not TRUE
```

### SessionInfo
```R
> sessionInfo()
R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Arch Linux

Matrix products: default
BLAS:   /usr/lib/libblas.so.3.10.0
LAPACK: /usr/lib/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C
 [9] LC_ADDRESS=C               LC_TELEPHONE=C
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets
[8] methods   base

other attached packages:
[1] GenomeInfoDb_1.28.4 IRanges_2.26.0      S4Vectors_0.30.2
[4] BiocGenerics_0.38.0

loaded via a namespace (and not attached):
[1] compiler_4.1.2         GenomeInfoDbData_1.2.6 RCurl_1.98-1.5
[4] bitops_1.0-7

```

**Comment by hpages** [comment]
See issues #30 and #31, and also https://support.bioconductor.org/p/9140106/ and https://support.bioconductor.org/p/9140106/ on the support site.

TLDR: This is fixed in BioC 3.14 so it's time to upgrade ;-)

**Comment by chrarnold** [comment]
This issue seems to have re-appeared, see #82
